### PR TITLE
feat(core): forward stdin to commands started via rust

### LINKED ISSUE #8269: Allow interactive commands through @nrwl/workspace:run-commands
## Description

Sometimes you just want to be able to wrap a CLI tool for a project without using a plugin. For example:

```
        "serve": {
            "executor": "@nrwl/workspace:run-commands",
            "options": {
                "commands": [
                    {
                        "command": "shopify app serve"
                    }
                ],
                "cwd": "apps/shopify-app"
            }
        }
```

When you run this, sometimes it prompts you

```
~/_code/shopify-example/apps/shopify-app feature/nx-serve-functions* ❯ shopify app serve                                                                                                                                                                        13:22:24
? Do you want to convert mystore.myshopify.com to a development store?
Doing this will allow you to install your app, but the store will become transfer-disabled.
Learn more: https://shopify.dev/tutorials/transfer-a-development-store-to-a-merchant#transfer-disabled-stores
 (Choose with ↑ ↓ ⏎)
> 1. yes
  2. no
```

But all I see is 

image

## Motivation

There are so many useful CLI tools out there which do not default to --non-interactive (or even have a non-interactive version) and when they prompt you can't actually see what is printed to the console. 

## Suggested Implementation

I think there are multiple execution layers, i'm not sure how to do this. Given a few pointers I could take a crack.

### PR DESCRIPTION
**Pull request by AgentEnder**
## Current Behavior
Commands ran via rust do not have any access to user input

## Expected Behavior
Commands ran via rust receive stdin

## Related Issue(s)
This removes the fix for https://github.com/nrwl/nx/issues/1666 - we believe the issue should no longer be relevant and it was causing the command runner to hang on MacOS - if it recurs, we will look into reapplying some version of the fix.

Fixes #8269

### CODE REVIEW

File: packages/nx/src/native/tests/command.spec.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { checkUnknownRules } from 'ajv/dist/compile/util';
```
Comment (Cammisuli) [inline packages/nx/src/native/tests/command.spec.ts:1]: I think this was added by accident. Lets remove it

File: packages/nx/src/native/tests/command.spec.ts [hunk 2]
```diff
@@ -26,23 +27,26 @@ describe('runCommand', () => {
...
     const childProcess = runCommand('echo "hello world"', process.cwd());
 
-    childProcess.onOutput((output) => {
-      expect(output.trim()).toEqual('hello world');
+    let output = '';
+    childProcess.onOutput((chunk) => {
+      output += chunk;
     });
-
+
```
Comment (Cammisuli) [inline packages/nx/src/native/tests/command.spec.ts:34]: this might fail prettier?

**Review by Cammisuli — CHANGES_REQUESTED** [review]

File: packages/nx/src/native/tests/command.spec.ts (hunk 1 above)
Comment (AgentEnder) [reply packages/nx/src/native/tests/command.spec.ts:1]: Yeah, thanks

File: packages/nx/src/native/tests/command.spec.ts (hunk 2 above)
Comment (AgentEnder) [reply packages/nx/src/native/tests/command.spec.ts:34]: 🤔 it pushed up without no verify, but I'll fix it while I remove the import.

Comment (AgentEnder) [reply packages/nx/src/native/tests/command.spec.ts]: Ah, there was a prettierignore rule hitting these spec files. I adjusted it.

**Review by Cammisuli — APPROVED** [review]
🎉

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**